src/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,57 @@
+.alert {
+  display: flex;
+  width: 100%;
+  margin-top: rem(24px);
+  margin-bottom: rem(32px);
+
+  animation: slide-in 0.2s ease-in;
+
+  border-radius: rem(12px);
+
+  background-color: rgba(37, 37, 37, 0.6);
+
+  color: #fff;
+  backdrop-filter: blur(1.698rem);
+  padding: 0;
```

Comment:
You can add the `Stylelint` extension to your Editor/IDE, and it will help you autoformat the code following the current rules.
This is the error when running `npm run style:lint`

![Extension__Stylelint_—_best-ic](https://user-images.githubusercontent.com/7344405/219543685-a22ff2e8-e151-414e-9b2e-f14bcaf1aefc.png)